app/liquid_tags/link_tag.rb

```diff
@@ -1,6 +1,7 @@
 class LinkTag < LiquidTagBase
   include ActionView::Helpers
   PARTIAL = "articles/liquid".freeze
+  REGISTRY_REGEXP = %r{#{URL.url}/[a-zA-Z0-9_]+/[a-zA-Z0-9-]+(/)?}
```

Comment:
Minor comments, non-blocking:

* `[a-zA-Z0-9_]`: This exist as the predefined character class `\w` ([source](https://ruby-doc.org/core-3.0.2/Regexp.html)):
    > /\w/ - A word character ([a-zA-Z0-9_])
* I don't think we need a capturing group around the final `/`, do we?
* Do we need to anchor the regex or is it ok for arbitrary text to appear after the optional last slash?

```suggestion
  REGISTRY_REGEXP = %r{#{URL.url}/\w]+/[a-zA-Z0-9-]+/?}
```

Reply:
> I don't think we need a capturing group around the final /, do we?

😅  I'm just learning regex due to this project, MK, but I'm learning lol. In my mind, to make the slash optional with the `?` I needed to put it in parentheses. But I see that was wrong now. I'm getting better 👍🏾

Reply:
@msarit That's fine, they can be intimidating at first. If you ever need help feel free to ping me on Slack, I wrote way too many over the years 😂